services/api-server/src/simcore_service_api_server/services/webserver.py

```diff
@@ -392,6 +392,20 @@ async def get_project_node_pricing_unit(
             data = Envelope[PricingUnitGet].parse_raw(response.text).data
             return data
 
+    async def put_project_node_pricing_plan_and_unit(
```

Comment:
at this interface I would avoid http verbs (except perhaps get) and rather focus on the context of this interface. For instance `PUT /projects/.../pricing-unit`'s handler (and `operation_id` in the webserver is `connect_pricing_unit_to_project_node`.

Reply:
replaced the name by `connect_pricing_unit_to_project_node`.